android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,5 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
```

Comment:
Commenting here as we can not commit on added files. Please remove the gradle-wrapper.jar file from the commit. We do not do this. Usually any dependency which is related from start building to running, should be downloaded unless there is nothing else to do. I believe this jar file will be downloaded when building the native app.

Reply:
The gradle-wrapper.jar file is a necessary component for building and running the app, and removing it may cause issues with the app's functionality. It is also possible that future changes to the app's code may require this file to be present.

Additionally, keeping the gradle-wrapper.jar file in the code repository can be useful for other developers who may be working on the project. They can use it to ensure that they have the correct version of the file, which may be required for building the app.

![Screenshot 2023-03-08 172955](https://user-images.githubusercontent.com/84832565/223708132-c4c447a7-81a0-4862-8e48-3ed143a56905.png)

Reply:
Please do following.

1. Remove the jar file (delete it).
2. Uninstall the app from mobile.
3. Rebuild the app

If you see the jar file that you removed in the exact location, we should not commit this. If not, let's commit.

Reply:
After following the suggested steps mentioned above, I attempted to build the application. However, when I removed the `gradle-wrapper.jar` file, I encountered an error, which can be seen in the following screenshot:

![Screenshot 2023-03-08 190815](https://user-images.githubusercontent.com/84832565/223727829-9449c654-293f-4f95-89ac-3cf5973ad36d.png)

Subsequently, I added the `gradle-wrapper.jar` file again and was able to successfully build the app.